### ISSUE TITLE
Switch a11y, css and toggle icon updates

### ISSUE THREAD

**Issue by mcoker** [patternfly/patternfly-react#1296]
From https://github.com/patternfly/patternfly-next/pull/1300

* changes `aria-label` to `aria-labelledby` on `.pf-c-switch__input`
    * although, @christiemolloy looking at the examples in core, I don't see elements with the `id`'s referenced in `aria-labelledby` in the switch examples. Can you clarify?
* ~CSS now toggles the switch label (on/off)~
* there is now an icon in the switch toggle when the switch has no label

**Comment by jgiardino** [comment]
Some parts of this issue are no longer valid, given this https://github.com/patternfly/patternfly-next/pull/1473 

Regarding aria-label vs aria-labelledby, ideally we would support either of these. For example, I should be able to reference another text string in my dom using `aria-labelledby` or I should be able to just define the text string using `aria-label`. I'd say this is more of an enhancement to the component we have, and not super critical.

**Comment by jgiardino** [comment]
@rebeccaalpert I grouped updates we could make to the switch into two groups: 1) what needs to be done so that the switch behaves as expected, and 2) what updates are nice to have. I would suggest for this issue we focus on what's needed, and defer updates that are nice to have until they're requested.

Also, for this issue any updates we make should not be breaking changes.

Some notes about the switch component:
- `input.pf-c-switch__input` can take either `aria-label` or `aria-labelledby` attribute, and since we currently support `aria-label` in the react component, it isn't super urgent to support `aria-labelledby`
- If https://github.com/patternfly/patternfly-next/pull/1473 gets merged, then the examples that show labels would have two labels with `.pf-c-switch__label` instead of one, and would be differentiated with `.pf-m-on` and `.pf-m-off` classes.
  - As part of this new example, core includes an `id` attribute on the `.pf-m-on` label which is used as the `aria-labelledby` value. If we add an `aria-labelledby` prop, the consumer should be able to control what the prop value is, and decide to include that `id` value as part of the value for `aria-labelledby`. As mentioned above, I'd say this can be a later enhancement, since they can provide that same string as part of the `aria-label` value.

**The updates:**
1. What needs to be done
    1. Update all examples with the label based on updates included in https://github.com/patternfly/patternfly-next/pull/1473. 
        - As part of these updates, the `.pf-m-on` and `.pf-m-off` classes should be included on the `.pf-c-switch__label` element.
        - Including `id` on `.pf-m-on` and `.pf-m-off` elements isn't necessary. See note below.
    2. Update all examples without a label to include the icon, as shown in core
2. Enhancements that we can include later
    1. Add support for `aria-labelledby`. Until this update is made, the need for `id` on the `.pf-c-switch__label.pf-m-on` isn't necessary.

Since https://github.com/patternfly/patternfly-next/pull/1473 is still open, it might make sense to break these down into separate PRs, and start with update 1.ii. listed above.

@mcoker Does this break down of what this issue should address seem right to you, or do you have any suggestions?

**Comment by rebeccaalpert** [comment]
Thanks. I added the icon and made the changes to the HTML with the exception of the id/aria-labelledby.

I won't be able to put up a PR for this until https://github.com/patternfly/patternfly-next/pull/1473 merges and is brought into PatternFly React since .pf-m-on and .pf-m-off aren't available to us yet.

**Comment by mcoker** [comment]
@jgiardino yep looks good to me, thanks for the clarification!

**Comment by rebeccaalpert** [comment]
Taking this off hold since the patternfly-next PR merged/pf-react was updated. Will rebase and put up a PR soon.
